src/settings.rs

```diff
@@ -548,6 +548,8 @@ pub enum InstallSettingsError {
     ),
     #[error("No supported init system found")]
     InitNotSupported,
+    #[error("Could not find the disk `/` resides on")]
```

Comment:
I intentionally didn't make this `target_os = "macos"` even though that's the only place it's used. I feel like it'd be kinda bad to guard this single error in a downstream crate with `target_os` e.g. if they decide to `match` on it at  any point.